wiki/Performance_points/de.md

```diff
@@ -0,0 +1,132 @@
+
+### Geschwindigkeit
+
+*Geschwindigkeit* ist ein Kernwert, der berücksichtigt, wie schnell in der Beatmap spielbare Elemente erscheinen.
+
+Beatmaps mit vielen spielbaren Elementen in kurzer Zeit haben einen hohen Geschwindigkeitswert. Das bedeutet, dass eine Beatmap mit einem höheren Geschwindigkeitswert schwerer ist und dadurch mehr pp belohnt.
+
+Das Resultat daraus ist, dass Mods wie [Double Time](/wiki/Game_modifier/Double_Time) und [Half Time](/wiki/Game_modifier/Half_Time) signifikant die Geschwindigkeit der Beatmap verändern, was der Perfomance Points Algorithmus berücksichtigt. Gleichermaßen beeinflussen diese Mods auch signifikant die Menge an pp die belohnt werden.
+
+### Genauigkeit
+
+*Siehe auch: [Genauigkeit](/wiki/Gameplay/Accuracy)*
+
+*Genauigkeit* ist ein Prozentwert, der die Fähigkeit eines Spielers bewertet, [Hit-Objekte](/wiki/Hit_object) zur richtigen Zeit zu treffen. Es ist auch ein Kernwert, um die individuelle Leistung eines Spielers auf einer Beatmap zu bewerten.
+
+Scores mit hoher Genauigkeit werden vom Algorithmus als beeindruckend betrachtet und geben deswegen hohe Punktzahlen. Eine [Full Combo](/wiki/Full_combo) mit 80 % Genauigkeit kann manchmal 2/3 so viel Wert haben als eine mit 95 % Genauigkeit. Dadurch, dass der Algorithmus stark auf Genauigkeit beruht, geben Mods wie Hidden, Hardrock und Flashlight beachtlich mehr pp für Scores mit hoher Genauigkeit.
+
+### Anstrengung
+
+*Anstrengung* ist ein Kernwert, der bewertet, wie oft und wie lang der Spieler Sektionen mit hoher Intensität in einer Beatmap spielen muss.
+
+Teile der Beatmap mit sehr hohe Geschwindigkeit oder schwierigen [Mustern](/wiki/Beatmap/Pattern) erhöhen den Anstrengungswert. Das heißt: Beatmaps mit mehr [Streams](/wiki/Beatmap/Pattern/Stream) oder schnellen Sprüngen, haben einen höheren Anstrengungswert und geben mehr pp.
+
+## FAQ
+
+### Wo kann ich meinen Performance-Rang sehen?
+
+**Die Performance-Points-Rangliste für alle Spieler kann auf der [Ranglistenseite](https://osu.ppy.sh/p/pp) gefunden werden.**
+
+Du kannst auch das Dropdown-Panel `Rankings` auf der alten Webseite benutzen und `Performance` auswählen.
+
+### Wie kann ich meinen Rang und insgesamte pp erhöhen?
+
+**Deine Performance wird überwiegend basierend auf deinen Scores auf einzelnen Beatmaps bewertet.**
+
+Der beste Weg, sich zu verbessern, besteht darin, gute Scores auf schwierigen Beatmaps zu setzen oder viele verschiedene Beatmaps zu spielen.
+
+Beachte folgende Tips:
+
+- Spiele effizient und finde den richtigen Spielstil für dich.
+- Fokussiere dich darauf, ein paar gute Scores zu setzen, anstatt mittelmäßige zu "farmen".
+- Versuche, deine Genauigkeit zu verbessern. Selbst 1 % macht einen großen Unterschied.
+- Versuche, hohe Combos zu erreichen. Eine Full Combo (FC) oder ein [SS](/wiki/Gameplay/Grade) geben enorm viele Punkte.
+
+### Warum habe ich nicht die volle Menge an pp bekommen von der Beatmap, die ich gespielt habe?
```

Comment:
```suggestion
### Warum habe ich von der Beatmap, die ich gespielt habe, nicht die volle Menge an pp bekommen ?
```

Reply:
Extraneous space before the question mark.

Reply:
Not anymore.